noxfile.py

```diff
@@ -489,6 +489,14 @@ def prerelease_deps(session, protobuf_implementation):
     ]
     session.install(*other_deps)
 
+    # Install spanner-graph-notebook, python-bigquery, and python-bigquery-storage
+    # from main to detect any potential breaking changes. For context, see:
+    # https://github.com/googleapis/python-bigquery-pandas/issues/854
+    session.install(
+        "https://github.com/cloudspannerecosystem/spanner-graph-notebook/archive/refs/heads/main.zip",
```

Comment:
Let's throw a `--pre` and `--upgrade` on there so that pip doesn't ignore these for already being installed.

```suggestion
        "--pre",
        "--upgrade",
        "https://github.com/cloudspannerecosystem/spanner-graph-notebook/archive/refs/heads/main.zip",
```